### ISSUE TITLE
unit conversion with sign

### ISSUE THREAD

**Issue by daniel-klein** [starsimhub/starsim#708]
```bash
self.pars.dur_pregnancy
ss.days(2, unit=day)

self.pars.dur_pregnancy.to('year')
ss.days(0.005479452054794521, unit=day, parent=year)

-self.pars.dur_pregnancy.to('year')
ss.days(-0.005479452054794521, unit=day, parent=year, values=-1.5012197410395947e-05)
```

Why does the negative sign change the value to -1.5012197410395947e-05? I had expected -0.005479452054794521.

**Comment by daniel-klein** [comment]
Workaround is `-float(self.pars.dur_pregnancy.to('year'))`

**Comment by cliffckerr** [comment]
Closed by #727
